common/src/test/java/io/pravega/common/concurrent/ExecutorServiceHelpersTests.java

```diff
@@ -85,4 +90,20 @@ public void testExecute() {
         Assert.assertNull("Unexpected exception set (rejected execution)", exceptionHolder.get());
         Assert.assertEquals("Unexpected number of finally runs (rejected execution)", 1, finallyCount.get());
     }
+
+    @Test
+    public void testSnapshot() {
+        @Cleanup("shutdown")
```

Comment:
Either you use `@Cleanup("shutdown")` or `ExecutorServiceHelpers.shutdown()`, but I don't think that is needed to use both, right?

Reply:
We need cleanup in case the test fails and we never get to shut it down. It's like a `finally` to make sure we "clean up".